sdk_extensions/zpages/src/main/java/io/opentelemetry/sdk/extensions/zpages/TraceConfigzZPageHandler.java

```diff
@@ -0,0 +1,343 @@
+  }
+
+  /**
+   * Emits the active tracing parameters table to the {@link PrintStream} {@code out}.
+   *
+   * @param out the {@link PrintStream} {@code out}.
+   */
+  private void emitActiveTable(PrintStream out) {
+    out.print("<table style=\"border-spacing: 0; border: 1px solid #363636;\">");
+    out.print("<tr class=\"bg-color\">");
+    out.print("<th class=\"header-text\"><b>Name</b></th>");
+    out.print("<th class=\"header-text border-left-white\"><b>Value</b></th>");
+    out.print("</tr>");
+    TraceConfigzActiveTableRow.builder()
+        .setPrintStream(out)
+        .setParamName("Sampler")
+        .setParamValue(this.tracerProvider.getActiveTraceConfig().getSampler().getDescription())
+        .setZebraStripeColor(ZEBRA_STRIPE_COLOR)
+        .setZebraStripe(false)
+        .build()
+        .emitHtml();
+    TraceConfigzActiveTableRow.builder()
+        .setPrintStream(out)
+        .setParamName("MaxNumOfAttributes")
+        .setParamValue(
+            Integer.toString(this.tracerProvider.getActiveTraceConfig().getMaxNumberOfAttributes()))
+        .setZebraStripeColor(ZEBRA_STRIPE_COLOR)
+        .setZebraStripe(true)
+        .build()
+        .emitHtml();
+    TraceConfigzActiveTableRow.builder()
+        .setPrintStream(out)
+        .setParamName("MaxNumOfEvents")
+        .setParamValue(
+            Integer.toString(this.tracerProvider.getActiveTraceConfig().getMaxNumberOfEvents()))
+        .setZebraStripeColor(ZEBRA_STRIPE_COLOR)
+        .setZebraStripe(false)
+        .build()
+        .emitHtml();
+    TraceConfigzActiveTableRow.builder()
+        .setPrintStream(out)
+        .setParamName("MaxNumOfLinks")
+        .setParamValue(
+            Integer.toString(this.tracerProvider.getActiveTraceConfig().getMaxNumberOfLinks()))
+        .setZebraStripeColor(ZEBRA_STRIPE_COLOR)
+        .setZebraStripe(true)
+        .build()
+        .emitHtml();
+    TraceConfigzActiveTableRow.builder()
+        .setPrintStream(out)
+        .setParamName("MaxNumOfAttributesPerEvent")
+        .setParamValue(
+            Integer.toString(
+                this.tracerProvider.getActiveTraceConfig().getMaxNumberOfAttributesPerEvent()))
+        .setZebraStripeColor(ZEBRA_STRIPE_COLOR)
+        .setZebraStripe(false)
+        .build()
+        .emitHtml();
+    TraceConfigzActiveTableRow.builder()
+        .setPrintStream(out)
+        .setParamName("MaxNumOfAttributesPerLink")
+        .setParamValue(
+            Integer.toString(
+                this.tracerProvider.getActiveTraceConfig().getMaxNumberOfAttributesPerLink()))
+        .setZebraStripeColor(ZEBRA_STRIPE_COLOR)
+        .setZebraStripe(true)
+        .build()
+        .emitHtml();
+    out.print("</table>");
+  }
+
+  /**
+   * Apply updated trace configuration through the tracerProvider based on query parameters.
+   *
+   * @param queryMap the map containing URL query parameters.
+   */
+  private void applyTraceConfig(Map<String, String> queryMap) {
```

Comment:
I think this method should be at the bottom of the class, rather than stuck here in the middle of the html rendering.